### DISCUSSION TITLE
Windows Release

### DISCUSSION THREAD

**Discussion by dylanschutte** [gitbutlerapp/gitbutler#2696] [Q&A] [answered]
Hey Is there a estimate on a windows release?

**Comment by krlvi** [comment, marked as answer]
Hey @RabieRabit! We are tracking the windows support in here https://github.com/gitbutlerapp/gitbutler/issues/2594
(and as a workaround KroniK907 https://github.com/gitbutlerapp/gitbutler/issues/2594#issuecomment-1944576256 made a really nice guide for setting GB up under WSL).

It's hard to give you a precise estimate right now, but I can say that this is definitely a top prio for us

**Reply by dylanschutte** [reply]
Thanks for letting me know so quick :D
